README.md

```diff
@@ -4,9 +4,11 @@ Demo to show a real-time trading dashboard for three different exchanges.
 
 The complete crypto-trading demo has two components:
 
-1. A stream application.
-2. A UI application written in Reactjs
+1. A backend built with Macrometa a Document Collection, multiple Streams, and a Stream Worker.
 
+If you haven't already configure the back end. Here's a link to the [Quick Start Guide](https://macrometa.com/docs/apps/crypto-trading).
```

Comment:
```suggestion
If you haven't already configure the backend, refer to the [Crypto Trading App](https://macrometa.com/docs/apps/crypto-trading).
```

Reply:
Whenever possible, the link text should match the title of what you are linking to.